lib/manageiq/cross_repo/runner.rb

```diff
@@ -3,9 +3,9 @@
 
 module ManageIQ::CrossRepo
   class Runner
-    attr_reader :test_repo, :core_repo, :gem_repos, :script_cmd
+    attr_reader :test_repo, :core_repo, :gem_repos, :script_cmd, :test_suite
 
-    def initialize(test_repo, repos, script_cmd = "")
+    def initialize(test_repo, repos, script_cmd = "", test_suite = "")
```

Comment:
Minor, but can test_suite come before script_cmd?  I'd think test_suite is more likely to be passed

Reply:
:+1: done